src/pymodaq/utils/leco/daq_move_LECODirector.py

```diff
@@ -143,22 +146,29 @@ def stop_motion(self) -> None:
         self.controller.stop_motion()
 
     # Methods accessible via remote calls
-    def _set_position_value(self, position: Union[str, float]) -> DataActuator:
-        if isinstance(position, str):
-            deserializer = DeSerializer.from_b64_string(position)
-            pos = deserializer.dwa_deserialization()
+    def _set_position_value(
+        self, position: Union[str, float, None], additional_payload=None
+    ) -> DataActuator:
+        if position:
+            if isinstance(position, str):
+                deserializer = DeSerializer.from_b64_string(position)
+                pos = deserializer.dwa_deserialization()
+            else:
+                pos = DataActuator(data=position)
+        elif additional_payload is not None:
+            pos = DeSerializer(additional_payload[0]).dwa_deserialization()
```

Comment:
what would be the additional payload?

Reply:
How the binary transfer works:
The first payload frame contains the JSON response.
The second (and further) frames contain binary objects. These are the `additional_payload`.

Therfore, the sending party sends the binary version of the PyMoDAQ object in the first additional payload frame and here it gets deserialized.